apps/api/src/domain/models/vocabulary-list/queries/__snapshots__/vocabulary-list-queries-fetch-many.e2e.spec.ts.snap

```diff
@@ -0,0 +1,71 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`when querying for a vocabulary list: fetch many when the user is unauthenticated when there is a vocabulary list with the given ID when the vocabulary list is published when each vocabulary list has one entry for a published term should return the correct vocabulary list view 1`] = `
+{
+  "entities": [
+    {
+      "actions": [],
```

Comment:
Do detail-scoped actions come through?

Reply:
The only way to verify this is to snapshot one of the admin responses instead of the pubilc response. I've done this for both query tests now.

Reply:
I've confirmed the snapshots now.